### DISCUSSION TITLE
Q. About Aqua's specifications / [質問] Aquaの仕様について

### DISCUSSION THREAD

**Discussion by fhrk-78** [e6nlaq/aqua#320] [Q&A] [answered]
Does Aqua have classes/functions? / Aquaにはクラス・関数は存在するのですか？
Also, does Aqua have a scope? / また、Aquaにはスコープも存在するのですか？

**Comment by e6nlaq** [comment, marked as answer]
全て**現在は**実装されていません。

クラスは未定ですが、関数・スコープは実装予定です。

なお、関数は`def`,スコープは`option`で制御するというのを考えています。

**Reply by fhrk-78** [reply]
了解です
今後に期待しています。ありがとうございます
